autoload/neomake/makers/ft/vim.vim

```diff
@@ -14,8 +14,11 @@ function! neomake#makers#ft#vim#vint() abort
 
     return {
         \ 'args': l:args,
-        \ 'errorformat': '%I%f:%l:%c:style_problem:%m,%f:%l:%c:%t%*[^:]:%m'
-        \ }
+        \ 'errorformat': '%I%f:%l:%c:style_problem:%m,%f:%l:%c:%t%*[^:]:%m',
+        \ 'postprocess': {
+        \   'fn': function('neomake#postprocess#GenericLengthPostprocess'),
+        \   'pattern': '\vUndefined variable: \zs([^ ]+)',
```

Comment:
> OTOH, using a list for the postprocess handlers felt dirrty already; as for the pattern, it could be easily be a group/ORed pattern.

Yeah, adding a loop for it wouldn't be ideal either.  Complicated patterns could be a list and joined into a string for the maker config.

I was making the comment wrt maintenance since I've had to craft some pretty gnarly patterns and it's sometimes tough keeping track of all the parenthesis (especially if you're trying to actually match parenthesis).  Hopefully these patterns don't get that unwieldy.